seedlet/assets/sass/blocks/utilities/_editor.scss

```diff
@@ -233,6 +233,19 @@
 	}
 }
 
+// Selects the post editor and not the site editor
+.block-editor-block-list__layout:not(.edit-site-block-editor__block-list){
```

Comment:
Is there a good reason to also not remove these margins in the site editor?

Reply:
See https://github.com/Automattic/themes/pull/2245#issuecomment-656380777

I was trying to keep the visual styling close to what already exists today. Instead of removing _all_ margins for all full-width blocks in the side editor, I used [more specific selectors](https://github.com/Automattic/themes/pull/2245/files#diff-4d8badb34f37a5e11b5e97c01b7db662R20-R22) in `seedlet-blocks` to remove margins from site content (and not site header / site footer)